mosql/__init__.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 
-__author__ = 'Mosky <http://mosky.tw>'
-__version__ = '0.10'
+VERSION = (0, 10,)
 
+__author__ = 'Mosky <http://mosky.tw>'
+__version__ = '.'.join(str(v) for v in VERSION)
```

Comment:
Because

    >>> import mosql
    >>> print(mosql.__version__)
    0.10
    >>> mosql.__version__ > '0.9'
    False
    >>> print(mosql.VERSION)
    (0, 10)
    >>> mosql.VERSION > (0, 9)
    True

I stole the idea from [Django]’s implementation, but there are probably many projects using this pattern.

[Django]: https://github.com/django/django/blob/master/django/__init__.py